Fluid/Ast/CaseStatement.cs

```diff
@@ -7,39 +8,38 @@ namespace Fluid.Ast
 {
     public class CaseStatement : TagStatement
     {
+        private readonly IReadOnlyList<WhenStatement> _whenStatements;
```

Comment:
field would ideally be of concrete  type, now foreach will allocate to get enumerator, if interface required then for might be a better option (used elsewhere because of this very reason)

Reply:
Missed some of them apparently